### PR TITLE
Add Register Option

### PR DESCRIPTION
**Pull request by jamestthompson3**
Fixes issue #125 

Changes: Added Template, View, Form, and URL for users to register with their email address.

Screenshots for the change:
screen shot 2017-10-19 at 11 53 00 pm

### CODE REVIEW

File: home/forms.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from .models import *
+
+class UserForm(forms.ModelForm):
+	password=forms.CharField(widget=forms.PasswordInput())
+
+	class Meta():
+		model=User
+		fields=('username','password','email')
+	def clean_username(self):
+		cleaned_data=super().clean()
```
Comment (divyam3897) [inline home/forms.py]: space before and after "="

File: home/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserProfileInfo',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('full_name', models.CharField(max_length=30)),
+                ('year', models.IntegerField()),
+                ('branch', models.CharField(max_length=30)),
```
Comment (divyam3897) [inline home/migrations/0001_initial.py]: instead of char field can you make it a dropbox in form to keep it uniform?

File: home/urls.py [hunk 3]
```diff
@@ -4,7 +4,8 @@
 app_name='home'
 urlpatterns = [
 url(r'^$', views.Home.as_view(), name = 'home'),
-url(r'^resources/', views.Resources.as_view(), name = 'resources')
+url(r'^resources/', views.Resources.as_view(), name = 'resources'),
```
Comment (divyam3897) [inline home/urls.py]: why this line change?

**Comment by jamestthompson3** [comment]
Now using PEP8 and improved UI

**Comment by divyam3897** [comment]
work in a branch and squash your commits into one.

**Pull request closed without merging**